sdks/java/io/clickhouse/src/main/java/org/apache/beam/sdk/io/clickhouse/ClickHouseWriter.java

```diff
@@ -39,19 +41,19 @@ public class ClickHouseWriter {
   private static final Instant EPOCH_INSTANT = new Instant(0L);
 
   @SuppressWarnings("unchecked")
-  static void writeNullableValue(
-      ClickHouseRowBinaryStream stream, ColumnType columnType, Object value) throws IOException {
+  static void writeNullableValue(ClickHouseOutputStream stream, ColumnType columnType, Object value)
+      throws IOException {
 
     if (value == null) {
-      stream.markNextNullable(true);
+      BinaryStreamUtils.writeNull(stream);
     } else {
-      stream.markNextNullable(false);
+      BinaryStreamUtils.writeNonNull(stream);
       writeValue(stream, columnType, value);
     }
   }
 
   @SuppressWarnings("unchecked")
-  static void writeValue(ClickHouseRowBinaryStream stream, ColumnType columnType, Object value)
+  static void writeValue(ClickHouseOutputStream stream, ColumnType columnType, Object value)
```

Comment:
At this point, I believe value is known to be non-null, so we should be able to mark it as such and remove the suppressed warning

Reply:
It is the same logic and i have only changed the API